### PR TITLE
[DOC] Update computing section

### PR DESCRIPTION
**Pull request by emdupre**
Closes https://github.com/poldracklab/poldracklab.github.io/issues/61.

Expands computing documentation to better serve as an internal resource.
cc @ebridge2

### CODE REVIEW

**Comment by emdupre** [comment]
This is now ready for review @poldrack !

File: labguide/computing/tacc.md
Comment (effigies) [inline labguide/computing/tacc.md:1]: This file should probably be removed if you don't plan to update it in this PR.

File: labguide/computing/sherlock/data-management.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+### Checking current quota limits
+
+There are several useful commands for assessing current storage usage on Sherlock:
+
+- `sh_quota` is a Sherlock-specific command that provides a general overview for all partitions a user has access to.
+  [Documentation is provided on their wiki](https://www.sherlock.stanford.edu/docs/storage/?h=sh_quota#checking-quotas).
+- `ncdu` provides an interactive explorer for current storage usage within a given directory.
+  This can be useful when identifying where quota usage is being allocated.
+  Sherlock [recommends running it in an interactive job](https://www.sherlock.stanford.edu/docs/storage/?h=ncdu#locating-large-directories).
```
Comment (effigies) [inline labguide/computing/sherlock/data-management.md:24]: This section is duplicating the access and resources page. Might be better just to link?

**Review by effigies — APPROVED** [review]
Pushed a few changes that I think should be uncontroversial.

Overall, this LGTM, but might be good to get someone with more day-to-day sherlock usage to give it a once-over.

File: labguide/computing/sherlock/data-management.md (hunk 1 above)
Comment (emdupre) [reply labguide/computing/sherlock/data-management.md:24]: Ah, thanks for catching ! I'll make an edit to move the sherlock wiki links over and then cross-link the sections 👍

**Comment by emdupre** [comment]
@ebridge2 and @effigies ; I've now streamlined the "access and resources" and "data management" sections. The diff looks bigger than it is, since now all sentences are on newlines !

I ended up not cross-linking the sections since they feel relatively self-contained now, but very happy for other opinions here. Thanks !

File: labguide/computing/sherlock/access-and-resources.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+When your home directory begins to get filled, it may be valuable to consider moving files to scratch directories, or group directories.
+`HOME`, `GROUP_HOME`, and `OAK` are persistent storage; `*SCRATCH` directories are subject to purging.
+
+Another useful tool is the disk usage command `du`.
+A useful and more interacrtive version of this command is `ncdu`.
+To use `ncdu`, add the following line to the bottom of your `~/.bash_profile`, which will load the `ncdu` module each time you log in to Sherlock:
+
+```bash
+$ ml system ncdu
+```
```
Comment (effigies) [inline labguide/computing/sherlock/access-and-resources.md:71]:
```suggestion
To use `ncdu`, add `ml system ncdu` to the bottom of your `~/.bash_profile`, which will load the `ncdu` module each time you log in to Sherlock.
You can add and access it with the following commands:

````bash
$ echo "ml system ncdu" >> ~/.bash_profile
$ source ~/.bash_profile
```
````

File: labguide/computing/sherlock/access-and-resources.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+In future login session, you can access the `ncdu` command via
```
Comment (effigies) [inline labguide/computing/sherlock/access-and-resources.md:73]:
```suggestion
In future login sessions, `ncdu` will be accessible without calling `source`.
Use the `ncdu` command via
```

**Comment by effigies** [comment]
Made a couple small suggestions. I'm +1 for merging, and things can always be improved.

**Comment by ebridge2** [comment]
Cool; I actually was concurrently reviewing and fixed that and added a few minor clarifications @effigies

**Pull request merged**